app/uk/gov/hmrc/hmrcemailrenderer/templates/TemplateLocator.scala

```diff
@@ -118,6 +118,15 @@ trait TemplateLocator {
   lazy val all: Seq[MessageTemplate] = templateGroups.values.flatten.toSeq
 
   def findTemplate(templateId: String): Option[MessageTemplate] =
-    all.find(_.templateId == templateId)
+    all.find(_.templateId == templateId) orElse  {
+      if (templateId.endsWith(TemplateLocator.WELSH_SUFFIX)) {
```

Comment:
do you need this if :) ? 
can't you just do or else all.find(_.templateId == templateId.stripSuffix(...)) ?

Reply:
yes, makes sense.